slycot/analysis.py

```diff
@@ -1831,75 +1901,85 @@ def ag08bd(l,n,m,p,A,E,B,C,D,equil='N',tol=0.0,ldwork=None):
+    m : int
+        The number of columns of matrix B.  m >= 0.
+    p : int
+        The number of rows of matrix C.  p >= 0.
+    A : (l, n) array_like
+        The leading l-by-n part of this array must
+        contain the state dynamics matrix A of the system.
+    E : (l, n) array_like
+        The leading l-by-n part of this array must
+        contain the descriptor matrix E of the system.
+    B : (l, m) array_like
+        The leading l-by-m part of this array must
+        contain the input/state matrix B of the system.
+    C : (p, n) array_like
+        The leading p-by-n part of this array must
+        contain the state/output matrix C of the system.
+    D : (p, m) array_like
+        The leading p-by-m part of this array must contain the
+        direct transmission matrix D of the system.
+    equil : {'S', 'N'}, optional
+        Specifies whether the user wishes to balance the system
+        matrix as follows:
+        = 'S':  Perform balancing (scaling);
+        = 'N':  Do not perform balancing.
+        Default is `N`
+    tol : float, optional
+        A tolerance used in rank decisions to determine the
+        effective rank, which is defined as the order of the
+        largest leading (or trailing) triangular submatrix in the
+        QR (or RQ) factorization with column (or row) pivoting
+        whose estimated condition number is less than 1/TOL.
+        If the user sets TOL <= 0, then default tolerances are
+        used instead, as follows: TOLDEF = L*N*EPS in TG01FD
+        (to determine the rank of E) and TOLDEF = (L+P)*(N+M)*EPS
+        in the rest, where EPS is the machine precision
+        (see LAPACK Library routine DLAMCH).  TOL < 1.
+        Default is 0.
+    ldwork : int, optional
+        The length of the cache array.
+        ldwork >= max( 4*(l,n), ldw ), if equil = 'S',
+        ldwork >= ldw,                 if equil = 'N', where
+        ldw = max(l+p,m+n)*(m+n) + max(1,5*max(l+p,m+n)).
+        For optimum performance ldwork should be larger.
+        Default is None.
+
+    Returns
+    -------
+    Af : (nfz, nfz) ndarray
+        the leading NFZ-by-NFZ part of this array
+        contains the matrix Af of the reduced pencil.
+    Ef : (nfz, nfz) ndarray
+        the leading NFZ-by-NFZ part of this array
+        contains the matrix Ef of the reduced pencil.
+    nrank : int
+        The normal rank of the system pencil.
+    niz : int
+        The number of infinite zeros.
+    infz : (n+1, ) ndarray
+        The leading DINFZ elements of infz contain information
+        on the infinite elementary divisors as follows:
+        the system has infz(i) infinite elementary divisors of
+        degree i in the Smith form, where i = 1,2,...,DINFZ.
+    kronr : (n+m+1, ) ndarray
+        The leading NKROR elements of this array contain the
+        right Kronecker (column) indices.
+    infe : (1+min(l+p,n+m), ) ndarray
+        The leading NINFE elements of infe contain the
```

Comment:
NINFE is hidden in the output

Reply:
SO are NKROR and DINFZ

Reply:
DINF, NKROR, NINFE, NKROL in the docstring have been deleted, ag08bd.